x-pack/plugins/global_search/README.md

```diff
@@ -0,0 +1,47 @@
+# Kibana GlobalSearch plugin
+
+The GlobalSearch plugin provides an easy way to search for various objects, such as applications
+or dashboards from the Kibana instance, from both server and client-side plugins
+
+## Consuming the globalSearch API
+
+```ts
+startDeps.globalSearch.find('some term').subscribe(
+  ({ results }) => {
```

Comment:
Optional: while this form is completely fine in the code when IDE gives you hints, it's not reader-friendly when used in the docs. What if we provide a verbose example:
```js
startDeps.globalSearch.find('some term').subscribe({
  next ({ results }) {
    addNewResultsToList(results);
  },
  error(){},
  complete() {
    showAsyncSearchIndicator(false);
  }
});
```